Dangerfile

```diff
@@ -0,0 +1,23 @@
+# Sometimes it's a README fix, or something like that - which isn't relevant for
+# including in a project's CHANGELOG for example
+declared_trivial = github.pr_title.include? "#trivial"
+has_pod_changes = !git.modified_files.grep(/MatomoTracker/).empty?
```

Comment:
MatomoTracker 🤔 🙃

Reply:
Uhm ... *cough cough* ... you got me there 😊

Reply:
Fixed this.